build/assets/file_type_terraform.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="64" height="64"><style><![CDATA[.C{fill-rule:evenodd}]]></style><g transform="matrix(.826767 0 0 .826767 8.416322 51.138164)" class="C"><path d="M39.1 15.557v-24.43l-21.16-12.22v24.43zm0-27.1l-21.16-12.22v-24.44l21.16 12.22z" fill="#5c4ee5"/><path d="M62.6-23.773v-24.44l-21.17 12.22v24.44z" fill="#4040b2"/><g fill="#5c4ee5"><path d="M15.62-25.193v-24.44l-21.16-12.22v24.44z"/><path d="M53.64-.483"/><path d="M17.94-21.093"/></g></g></svg>
```

Comment:
What's `![CDATA[.C{fill-rule:evenodd}]]` doing in your SVG file? That looks a bit odd, do we need to clean it up a bit perhaps? I like to see a properly indented svg file with a style block that's easy to edit later on.

Reply:
Cleaned up SVG.